pkg/webhooks/validator/validator.go

```diff
@@ -390,6 +398,16 @@ func (wh *WebhookHandler) validateMediatedDeviceTypes(hc *v1beta1.HyperConverged
 	return nil
 }
 
+func (wh *WebhookHandler) validateFeatureGates(hc *v1beta1.HyperConverged) error {
+	// reject the EnableManagedTenantQuota on a single node cluster
+	if !hcoutil.GetClusterInfo().IsInfrastructureHighlyAvailable() {
+		if hc.Spec.FeatureGates.EnableManagedTenantQuota != nil && *hc.Spec.FeatureGates.EnableManagedTenantQuota {
+			return fmt.Errorf("the EnableManagedTenantQuota feature gate is not supported on non highly available cluster")
```

Comment:
I think it would be better not to use double-negatives, e.g.
```suggestion
			return fmt.Errorf("the EnableManagedTenantQuota feature gate is supported only on highly available clusters")
```